src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/config/CacheConfig.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.iacaseapi.infrastructure.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import java.util.concurrent.TimeUnit;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+class CacheConfig {
+
+    @Bean
+    public CacheManager cacheManager() {
+        Caffeine<Object, Object> caffeineBuilder = Caffeine.newBuilder()
+            .maximumSize(500)
+            .expireAfterAccess(1, TimeUnit.MINUTES);

```

Comment:
Don't think we need to only expire after access, we expire everything that we write after a fixed interval and that interval can and should outlive the duration of Retries - couple of minutes would be good enough!